samples-generator/bin/3d-tiles-samples-generator.ts

```diff
@@ -5,13 +5,14 @@ var Cesium = require('cesium');
 var fsExtra = require('fs-extra');
```

Comment:
`CESIUM_3dtiles_feature_hierarchy` should be listed in `extensionsUsed`

Reply:
```
          "attributes": {
            "POSITION": 0,
            "NORMAL": 1,
            "_BATCHID": 2
          },
```

The `use3dTilesNext` flag must not be getting propagated correctly to `createGltf`. `_BATCHID` should be `_FEATURE_ID_0`.

Reply:
The hierarchy extension should be contained in an individual feature table, like

```
  "extensions": {
    "CESIUM_3dtiles_feature_metadata": {
      "featureTables": [
        {
          "properties": {
            "height": {
              "accessor": 4
            },
            "area": {
              "accessor": 5
            }
          },
          "featureCount": 30,
          "extensions": {
            "CESIUM_3dtiles_feature_hierarchy": {
            }
          }
        }
      ]
    }
  }
}
```